src/function/table/arrow_conversion.cpp

```diff
@@ -1340,17 +1340,31 @@ static void ColumnArrowToDuckDBDictionary(Vector &vector, ArrowArray &array, Arr
 }
 
 void ArrowTableFunction::ArrowToDuckDB(ArrowScanLocalState &scan_state, const arrow_column_map_t &arrow_convert_data,
-                                       DataChunk &output, idx_t start, bool arrow_scan_is_projected) {
+                                       DataChunk &output, idx_t start, bool arrow_scan_is_projected,
+                                       idx_t rowid_column_index) {
```

Comment:
This code does not seem to be called anywhere - meaning at the very least it's not tested. Do we need this?